semantic_conventions/trace/http.yaml

```
@@ -117,6 +117,10 @@ groups:
         type: int
         brief: >
           The ordinal number of request re-sending attempt.
+        note: >
+          The retry counter SHOULD be updated each time an HTTP request gets re-sent by the client, regardless of what
```

```suggestion
          The retry count SHOULD be updated each time an HTTP request gets re-sent by the client, regardless of what
```
Not sure what is best, counter as in the "counter variable" or the "count concept". I feel `counter` is already giving impl details in the spec.

---

True; and if the attribute name uses "count" then I think it's only reasonable to use "count" everywhere. Will change, thanks!